### PR TITLE
add optional api server host

### PR DESCRIPTION
**Pull request by automatedops**
Added `api-server-host` optional attribute. So that server can use `localhost` instead of `fqdn` to call API.

### CODE REVIEW

File: src/main/java/in/ashwanthkumar/gocd/slack/ruleset/Rules.java [hunk 1]
```diff
@@ -92,6 +93,19 @@ public Rules setGoServerHost(String goServerHost) {
         return this;
     }
 
+
+    public String getGoAPIServerHost() {
+        if (goAPIServerHost != null) {
```
Comment (ashwanthkumar) [inline src/main/java/in/ashwanthkumar/gocd/slack/ruleset/Rules.java]: Will it help if we can use `StringUtils.IsNotBlank` instead of just a null check?

**Comment by ashwanthkumar** [comment]
@WeiBanjo Thank you so much for the contribution :+1: I just have 1 minor comment, if you can fix that - I'll merge this in.

**Comment by ashwanthkumar** [comment]
Fixes #25

**Comment by automatedops** [comment]
Updated PR to use `StringUtils.isNotEmpty`.

**Pull request merged**